### PR TITLE
added option to upload file from system

### PR DESCRIPTION
**Pull request by Aravind22**
Changed URL image upload to file upload

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,8 +47,9 @@
         <div class="output">
             <textarea></textarea>
             <div class="imgform">
-                <input id="imgfield" type="text" placeholder="Paste image url here"></input>
-                <button id="loadimg">Load Image</button>
+                <!-- <input id="imgfield" type="text" placeholder="Paste image url here"></input> -->
+                <!-- <button id="loadimg">Load Image</button> -->
+                <input id="browseimg" type="file" name="bg" accept="image/*" onchange="loadbg(this)"></button>
```
Comment (choyiny) [inline index.html]: I would move the `onchange` attribute to a jQuery listener.

File: assets/js/main.js [hunk 2]
```diff
@@ -84,6 +84,16 @@ function setActiveTool(tool) {
     $("#" + tool + "tool").addClass("active-tool");
 }
 
+function loadbg(input){
+    if(input.files[0]){
```
Comment (choyiny) [inline assets/js/main.js]: Fix jslint errors

**Review by choyiny — CHANGES_REQUESTED** [review]
Thanks for your contribution! Small changes.

**Comment by Aravind22** [comment]
Please check the new commit @choyiny

**Review by choyiny — APPROVED** [review]

**Pull request merged**